README.md

```
@@ -6,8 +6,10 @@
 
   <p>Kubernetes Native Health Check Platform</p>
   <p>
-    <a href="https://github.com/flanksource/canary-checker/actions"><img src="https://github.com/flanksource/canary-checker/workflows/Test/badge.svg"></a>
-    <a href="https://goreportcard.com/report/github.com/flanksource/canary-checker"><img src="https://goreportcard.com/badge/github.com/flanksource/canary-checker"></a>
+    <a href="https://github.com/flanksource/canary-checker/actions/workflows/test.yml"><img src="https://github.com/flanksource/canary-checker/actions/workflows/test.yml/badge.svg?branch=master"></a>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)